components/app-core/backend/repository/console_config/psql_console_config.go

```diff
@@ -125,7 +125,19 @@ func (c *ConsoleConfigRepository) IsInitialised() (bool, error) {
 
 	rowCount, err := c.getTableCount()
 	if err != nil {
-		return false, err
+		for strings.Contains(err.Error(), "does not exist") {
+			// Schema isn't initialised yet. Wait a few secs and retry
+			log.Warnf("It appears schema isn't initialised yet, sleeping and trying again %s", err)
+			time.Sleep(1 * time.Second)
+			rowCount, err = c.getTableCount()
+			if err == nil {
+				break
+			}
+		}
+		if err != nil {
```

Comment:
Can this ever be true? err must be nil to get into the for block - then it only exits if err is nil, so I think this check will never be true